polygon-validator-signer-balance-bot/src/agent.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { FindingType, FindingSeverity, Finding, ethers, HandleBlock, createBlockEvent } from "forta-agent";
+import { provideHandleTransaction } from "./agent";
+import { POLYGON_VALIDATOR_SIGNER_ADDRESS } from "./constants";
+
+// let MINIMUM_THRESHOLD_BIG = ethers.BigNumber.from(MINIMUM_THRESHOLD)
+describe("POLYGON-VALIDATOR-SIGNER BOT TEST SUITE", () => {
```

Comment:
To piggyback off of Vasilis comments, the additional tests that would be great to see are:
* Returns only one finding when the balance is below the threshold for more than two blocks
    * Could do this by having two different instances of passing a block to the block handler, and checking the returned amount of findings after each. The first should return one finding, the second should return nothing, despite having the same balance as the first
 * Same as above, but for balance above a threshold.
    * This would entail three blocks. The first would need to be above the threshold. The next block will need to drop below the threshold, creating a finding. Then the last block should remain below the threshold, and not create a finding.
  * Similar as the second, but the third block should come back up and pass the threshold to create a second finding.
  * The bot should not create a finding for an address that is not the Polygon Signer address
    * For this, you could pick out one of the above scenarios, but switch out the address whose balance changes